test/tools/yulInterpreter/EwasmBuiltinInterpreter.cpp

```diff
@@ -54,9 +54,6 @@ void copyZeroExtended(
 /// Count leading zeros for uint64
```

Comment:
I think rather than moving the check around you should just comment here and specify how it treats zero.

Reply:
But maybe you can convince me otherwise, if you elaborate on the reason a bit :-).

Reply:
Especially since without having the special zero case in here, this will loop infinitely if you call it with zero...

Reply:
Since you say you want to use builtins instead of implementing this: We can do this, but I'd still do it like this:
```
uint64_t clz (uint64_t _v) { return _v ? __builtin_clzll(_v) : 64; }
```

Reply:
The problem with that is that we'll need ``#ifdef``s for all compiler versions, because it's not named the same everywhere, and probably *still* a fallback to a version we implement ourselves. Not sure that's worth it.

Reply:
> I think rather than moving the check around you should just comment here and specify how it treats zero.

How about `assert(_v != 0)`? `yulAssert()` is probably too heavy if this is performance critical (and it surely is if we're considering using compiler builtins) but this is exactly the case where you would use plain `assert()`.